flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": [ "i386", "x86_64", "aarch64" ]
```

Comment:
i386 isn't supported anymore

Reply:
Whoops, I copied that from Games repo 👀️